osidb/signals.py

```diff
@@ -72,3 +72,25 @@ def auto_create_profile(sender, instance, created, **kwargs):
 @receiver(pre_save, sender=AffectCVSS)
 def populate_cvss_score(sender, instance, **kwargs):
     instance.score = instance.cvss_object.scores()[0]
+
+
+@receiver(post_save, sender=Flaw)
```

Comment:
What are the advantages of using django signals ? 

It is my understanding that this is just running in django/python eg. if a row is updated in the database outside of django then we need to manually update to ensure local_updated_dt is updated. Perhaps that is ok ... maybe comments saying so here would provide hints to our future selves.

Reply:
As opposed to what? pg triggers?

You are correct in that this can be bypassed by using a few Django ORM methods (e.g. `update`, `bulk_update`, ...) but that also bypasses all the `save()` logic that we already have, so as a rule of thumb bypassing the ORM is a bad idea and in practice we only do it during hefty data migrations which are rare.

I can add comments, but it essentially applies to all signals

Reply:
I do not see why we would use triggers here either eg. why not overload save() and call super.save() on the model itself ? It is quite possible I am django ignorant and missing some obvious benefit. Maybe I should learn how django signals are implemented as well ;)

As for bypassing ... was mostly thinking of raw SQL against the database (outside of django). I guess I am never going to feel comfortable about using an ORM if it always forces one to never use the database (out of safety concerns)!

Reply:
`save` contains a lot of custom logic itself already and doesn't help in cases that bypass the ORM anyways, this is a cleaner, self-contained approach although I admit it can be "hidden" or "magic", but I'm just using the ORM's features.

Furthermore, with the latest change the use of signals is required, as there is no way (to my knowledge) to hook into a ManyToMany field's add/remove/etc. methods, and using these does not call `save`